src/WebUI/WorkLoads/WorkloadsPivot.tsx

```diff
@@ -89,18 +86,18 @@ export class WorkloadsPivot extends BaseComponent<IWorkloadsPivotProps, IWorkloa
 
     private _getContent(): JSX.Element {
         return (this.state.workloadResourceSize === 0 ? this._getZeroData() :
-            <div>
+            <>
```

Comment:
what are these empty brackets for ?

Reply:
https://reactjs.org/docs/fragments.html
this is short form of fragment in react, with this we may not need to add unnecessary div if a component is split in to more than one component.